primitiv/random.h

```diff
@@ -50,7 +51,19 @@ class DefaultRandomizer : mixins::Nonmovable<DefaultRandomizer> {
     std::uniform_real_distribution<float> dist(lower, upper);
     for (std::size_t i = 0; i < size; ++i) {
       const float x = dist(rng_);
+#ifdef PRIMITIV_MAYBE_FPMATH_X87
+      /**
+       * NOTE(vbkaisetsu):
+       * In x87 environments, a random variable `x` is stored in
+       * the 80 bit register. Almost values are not matched to the lower
+       * bound, but some of these values will be rounded to the lower
+       * bound after the data is returned from this function.
+       * This code cares for the internal representation of 80 bit floats.
+       */
+      data[i] = x < std::nextafter(lower, upper) ? upper : x;
```

Comment:
This code looks generally correct and the compiler switch and the comment are not necessary.

Reply:
Use a const variable out of the for-loop instead of directly describing `std::nextafter(lower, upper)`.